cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -175,6 +190,39 @@ public void execute() {
                 return;
             }
             isSingleFile = true;
+        } else if (isModule == true) {
```

Comment:
we can simply use "isModule" instead of "isModule == true"

Reply:
Corrected to `isModule`